.github/ISSUE_TEMPLATE/syntax_update.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Web IDL Syntax Change
+about: Let us know if there are changes in WebIDL syntax
+title: ""
```

Comment:
Maybe this should start with BREAKING CHANGE:?

Reply:
I was thinking someone might skip the title if it is prefilled with something. WDYT?

Reply:
I think a title ending with a colon explicitly requires adding more details. If anyone misses it, that means it's possibly just a spam.